tests/functional/UseCase/Bucket/ShareLinkBucketTest.php

```diff
@@ -384,6 +384,11 @@ public function testShareUnshareLinkedBucket(): void
         $targetDataset = $targetProjectBqClient->dataset($linkedBucketSchemaName);
         $this->assertTrue($targetDataset->exists());
         $testTableBefore = $targetDataset->table(self::TESTTABLE_AFTER_NAME);
-        $this->assertTrue($testTableBefore->exists());
+
+        // after unshare the table is not available
+        // in connection you can't just unshare a bucket that is lined up first so this is an edge case
+        // handled in connection
+        $this->expectException(BadRequestException::class);
```

Comment:
Zas to zacalo padat, a ked som vratil to co tam bolo povodne tak to zacalo fungovat.

Reply:
Tomu nějak nerozumím :(